src/main/java/duke/task/todo.java

```diff
@@ -0,0 +1,16 @@
+package duke.task;
+
+public class todo extends Task{
```

Comment:
Cross-check the coding standard to see if your class names are following the given convention.